flyteadmin/auth/authzserver/metadata_provider.go

```diff
@@ -71,10 +79,11 @@ func (s OAuth2MetadataProvider) GetOAuth2Metadata(ctx context.Context, r *servic
 			}
 			httpClient.Transport = transport
 		}
-
-		response, err := httpClient.Get(externalMetadataURL.String())
+		logger.Info(ctx, "retryAttempts: %v retryDuration: %v", s.cfg.AppAuth.ExternalAuthServer.RetryAttempts, s.cfg.AppAuth.ExternalAuthServer.RetryDelayMilliseconds)
```

Comment:
nit can we remove this before merging?

Reply:
Absolutely! will leave as unresolved as a reminder